aiida_orca/parsers/__init__.py

```diff
@@ -88,10 +88,12 @@ def _remove_nan(parsed_dictionary: dict) -> dict:
             relaxed_structure = StructureData(ase=ase_structure)
             self.out('relaxed_structure', relaxed_structure)
 
-        pt = PeriodicTable()  # pylint: disable=invalid-name
-
-        output_dict['elements'] = [pt.element[Z] for Z in output_dict['atomnos'].tolist()]
+        if output_dict.get('atomnos') is not None:
```

Comment:
```suggestion
        if output_dict.get('atomnos', None) is not None:
```
for safety?

Reply:
I checked the Python docs and dict.get() should return None by default if the key is missing.

Reply:
Wow, you are absolutely right. I was almost certain this wasn't the case. What the hell 😄